microsoft-edge/extensions-chromium/publish/add-ons-curation.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Curation and review process for extensions at Microsoft Edge Add-ons
+description: How an extension that's submitted at Microsoft Edge Add-ons is curated and reviewed by the Microsoft Edge extensions team.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: extensions
+ms.date: 10/13/2023
+---
+# Curation and review process for extensions at Microsoft Edge Add-ons
+
+The Microsoft Edge extensions team curates the experience at [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/) so that users can find all the items to customize their browser in one place, such as themes, extensions, and apps.
+
+Available _items_ to customize the user's browser includes themes, extensions, and apps.
+
+Before installing an item from [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/), users can learn more about the functionality, data permissions, and publisher by viewing the item's description and listing.
+
+
+<!-- ====================================================================== -->
+## How items are organized at Microsoft Edge Add-ons
+
+Items at [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/), such as themes, extensions, and apps, are organized to help users discover the ones that are relevant to their interests.
+
+Users can discover extensions they're interested in at [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/) in several ways, including:
+* The home page.
+* The Search bar.
+* Curated collections of extensions at [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/).
+* Categories of extensions at [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/).
```

Comment:
```suggestion
Users can discover extensions they're interested in at [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/) in several ways, including:

* The home page.
* The search bar.
* Curated collections of extensions.
* Categories of extensions.
```

I don't think we need to repeat "at Microsoft Edge Add-ons" for the last two items in the list. It's already in the lead-in sentence.